### DISCUSSION TITLE
How to add an image link with Markdown?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#837] [Ask a question] [answered]
Is it possible to add an image with a hyperlink using Yellow and markdown?

This methods only displays text, not an image:

```[![image alt text](image url link)](anchor link)```

**Comment by IngoBelka** [comment]
This should work:

`[[image example.png "Example-Image-Link" - 50%]"Example-Image-Link"](/Example-Link/) `

**Comment by glow12121** [comment]
Thank you! `! [[image picture.png "Comment"]](http://link.com)` has worked for me.

A side-note. For some reason, there was a barely visible grayish square around the image. I examined the source code and found that it was due to `.notice1` class. Not sure why it has to do with an `image`.

```
.content .notice1 {
    margin: 1em 0;
    padding: 10px 1em;
    background-color: #fffbf0;
    border-radius: 3px;
}
```

**Reply by annaesvensson** [reply]
You are 90% there, just remove the extra `!` character. An exclamation mark and a space at the beginning of a line would indicate the making of a [general block element](https://github.com/annaesvensson/yellow-markdown), hence the extra CSS class. Use the following instead:

`[[image photo.jpg "This is an example image"]](https://datenstrom.se)`

**Reply by glow12121** [reply]
Great, it works, thank you. It's too bad I cannot mark you sub-comment as the reply.

**Reply by annaesvensson** [reply]
Maybe just maybe, mark it as unanswered and you get a new answer from our community. 😄

**Reply by glow12121** [reply]
Already tried this but it seems that a sub-comment cannot be an answer. I've made the thread unanswered again.

**Comment by annaesvensson** [comment, marked as answer]
This should work, Markdown and HTML:

`[[image photo.jpg "This is an example image"]](https://datenstrom.se)`

`<a href="https://datenstrom.se"><img src="/media/images/photo.jpg" width="1000" height="400" alt="This is an example image" title="This is an example image" /></a>`
